src/pages/projectSettings/tabs/ContainersTab/ContainerSizeRow.tsx

```diff
@@ -0,0 +1,27 @@
+import styled from "@emotion/styled";
+import { ObjectFieldTemplateProps } from "@rjsf/core";
+import { size } from "constants/tokens";
+import { form } from "../utils";
+
+const { getFields } = form;
+
+export const ContainerSizeRow: React.VFC<
+  Pick<ObjectFieldTemplateProps, "formData" | "properties" | "uiSchema">
+> = ({ formData, properties }) => {
+  const [name, memoryMb, cpu] = getFields(properties, formData.isDisabled);
+
+  return (
+    <RowContainer data-cy="container-size-row">
+      {name}
+      {memoryMb}
+      {cpu}
```

Comment:
[nit] Since we don't need to do any special handling for each field it may be more futureproof to just map the props returned by getFields